suite-common/wallet-utils/src/stakingUtils.ts

```diff
@@ -41,6 +44,8 @@ export const getAccountTotalStakingBalance = (account: Account) => {
             return getEthAccountTotalStakingBalance(account);
         case 'solana':
             return getSolAccountTotalStakingBalance(account);
+        case 'cardano':
```

Comment:
@izmy can we reuse some staking type which says that there is staking on specific network? so that we do not miss it next time? Thanks

Reply:
Maybe StakingNetworkSymbol but extend it by cardano 😀